hashintel/hash
# Table Block Cleanup

## 🌟 What is the purpose of this PR?

This PR solves few issues of table block.

## 🔍 What does this change?

- Add example data to `package.json`
- Fix some css problems (svg icons not working, some values without `px`)

## ❓ How to test this?

1.  Checkout the branch / view the deployment
1.  Start table block
1. Confirm the fixes listed above
